### ISSUE TITLE
Feature request - Resume/Cover letter in jobsync

### ISSUE THREAD

**Issue by LayersOfAbstraction** [Gsync/jobsync#11]
Hi Khuram,

If possible would really like if document uploading can be supported for the https://demo.jobsync.ca/dashboard/myjobs section so I can store tailored resumes there.

**Comment by Gsync** [comment]
Hi,

Yes it can be done, wouldn't it be better to upload file at resume section and link when new job is applied?

**Comment by LayersOfAbstraction** [comment]
You mean in a section such as this?
https://demo.jobsync.ca/dashboard/profile/resume/be634c5a-fa73-4452-af1a-a099a33caea9

As in upload and link it to the job in something like a linked dropdown box or something? (FYI haven't looked at the code much). 

I don't think it would be better for us. Doing so would force us job seekers to abandon our favorite resume creation tools when all we want to do is export our resume and cover letter documents into myjobs section to keep track.

In time it may be more feasible for us to instead link the uploaded resume and cover letter from the myjobs screen to the selected profile/resume section.

**Comment by Gsync** [comment]
Problem is user might upload duplicate resumes for multiple jobs, so its better to upload once and link to applied job, system might be able to parse some details from uploaded file.

**Comment by LayersOfAbstraction** [comment]
Yes I know that could happen, but you could also lock that feature in the jobs screen and present a red coloured warning stating: 

> WARNING: uploading documents here may result in duplicate cover letters and resumes in future should you upload them in the Profile Screen. If you wish to continue this anyway, please enable this by ticking the **Enable documents attachments in Jobs** checkbox in the Settings menu. 

Implementing such a feature saves me from navigating to different screens just to store these documents. It would allow me to not have to store all the info about each job in an Excel spreadsheet because I still have to manually: 

1. Open the Windows File Explorer. 
2. Create the company folder. 
3. Create a job subfolder. 
4. Create the resume and cover letter in that subfolder.

This is my current file structure and as you can see it is an absolute pain to link each document.

Job tracker document folder
│
├── Google LLC
|      |--- Software Developer 
│   ├── John Doe - Resume.pdf
│   ├── John Doe - Cover Letter.pdf

I could develop the end user documentation for this idea, in a separate branch, down the track if you did this. I already have a paid resume.io subscription with AI integration. It helps if jobsync can accommodate me keeping track of our job submissions. 

Others could then more likely latter help improve the resume/cover letter features.

**Comment by Gsync** [comment]
Yes, we could do something like this to select resume from list or upload new. I will try that over the weekend.

**Comment by Gsync** [comment]
Ability to add/upload resume file and link it to add/edit job as has been added, its now included in the main branch of the app. Cover letter would be added later.

**Comment by ScottKFraley** [comment]
Oh, bummer. But thanks for letting me know. 

As I also mentioned, I've got zero time for this at this time.
